packages/showcases/svelte/stories/components/indicators/VtmnProgressbar/VtmnProgressbar.stories.svelte

```diff
@@ -34,6 +34,7 @@
       control: {
         type: 'text',
       },
+      defaultValue: 'Loading',
```

Comment:
Always "loading" by default ?
Should be required ?

Reply:
Yes, it should have a label text for accessiblity conformity.

Reply:
`defaultValue` on the story defined the value on the component (on the svelte file for example) have "loading" by default. You need to set the `label` required true.

I you want to set the loading value by default, you should set the `arg` on the Story.